extensions/wikia/EditPreview/js/preview.js

```diff
@@ -60,6 +60,11 @@ define('wikia.preview', [
 					'overflow-x': 'hidden'
 				});
 
+				// SUS-126: prevent page scrolling on large pages breaking the classic editor
+				$(document.body).css({
```

Comment:
Please use `$('body').css('overflow-y', 'hidden')` <- both lookup and function will be faster